docs/tutorials/schrödinger_equation.rst

```diff
@@ -0,0 +1,276 @@
+
+.. jupyter-execute::
+
+    alias = numpy_alias()
+    unp = alias()
+
+3. Define the equation using :class:`.Alias`
+--------------------------------------------
+
+We solve the Schrödinger equation using the Runge-Kutta method in this tutorial.
+The Schrödinger equation is written as
+
+.. math:: \frac{\partial\psi}{\partial t} = -i * H \psi
+
+The equation we are solving represents a qubit's state as a two-level system. The Hamiltonian is
+
+.. math:: H = \frac{1}{2} \times 2 \pi \nu_z {Z} + 2 \pi \nu_x \cos(2 \pi \nu_d t){X},
+
+where :math:`\{X,Y,Z\}` are the Pauli matrices.
+
+We can express the right-hand side (RHS) function of this equation as follows:
+
+.. jupyter-execute::
+
+    def rhs(t,y,Z,X):
+        return unp.matmul(-1j * (5 * Z -  unp.cos(10 * t) * X ),y)
+
+We can utilize :class:`.Alias` intuitively just like `NumPy <https://numpy.org/>`_, `JAX <https://github.com/google/jax>`_, and so on.
+This generic code provided by Arraylias includes ``unp.matmul()``, which automatically select the appropriate module
+based on the input type.
+
+We eventually want to find the probability of existence of this qubit state, so we will prepare the following function.
+
+.. jupyter-execute::
+
+    def state_probabilities(state):
+        probabilities = unp.abs(state) ** 2
+        return probabilities / unp.sum(probabilities)
+
+4. Solve the equation using existing solvers
+--------------------------------------------
+
+First, we solve the equation by using Numpy as the input and ``scipy.integrate.solve_ivp`` as a solver.
+We define the initial state, the time span for the simulation, and time point we want to simulate.
+
+.. jupyter-execute::
+
+    Z = np.array([[1,0],[0,-1]])
+    X = np.array([[0,1],[1,0]])
+    init_state = np.array([1. + 0j,0. + 0j])
+
+    t_span = [0,(N-1) * dt]
+    T = np.linspace(0,(N-1) * dt,N)
+
+We solve by using ``scipy.integrate.solve_ivp`` and plot the probabilities of each state.
+
+.. jupyter-execute::
+
+    sol = solve_ivp(rhs,t_span,init_state,args=(Z,X),method='RK45',t_eval=T)
+    probabilities = state_probabilities(sol.y)
+
+    plt.plot(sol.t, probabilities[0], label="0")
+    plt.plot(sol.t, probabilities[1], label="1")
+    plt.xlabel('T')
+    plt.ylabel('Probability')
+    plt.legend()
+    plt.show()
+
+
+Second, we solve the equation by using Jax.array as the input and ``jax.experimental.ode.odeint`` as a solver.
+
+.. jupyter-execute::
+
+    Z = jnp.array([[1,0],[0,-1]])
+    X = jnp.array([[0,1],[1,0]])
+    init_state = jnp.array([1. + 0j,0. + 0j])
+
+    t_span = [0,(N-1) * dt]
+    T = jnp.linspace(0,(N-1) * dt,N)
+
+    sol = odeint(lambda y,t: rhs(t,y,Z,X),init_state,T)
+    probabilities = state_probabilities(sol.T)
+    plt.plot(T, probabilities[0], label="0")
+    plt.plot(T, probabilities[1], label="1")
+    plt.xlabel('T')
+    plt.ylabel('Probability')
+    plt.legend()
+    plt.show()
+
+
+
+5. Register custom functions using :meth:`.Alias.register_function`
```

Comment:
Maybe change this to 
```suggestion
5. Register custom solvers using :meth:`.Alias.register_function`
```

Reply:
I fixed at https://github.com/Qiskit-Extensions/arraylias/pull/25/commits/53eee4341415b76857707888dd7a1d1cf73d19c9 .